internal/utils/blocks_test.go

```diff
@@ -1,13 +1,13 @@
 // Copyright IBM Corp. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package httputils_test
+package utils_test
```

Comment:
Why do we need new package here?

Reply:
Not related to this pr exactly, but you change package name in any case

Reply:
I like black box testing to check exported methods. 
Anyway, I changed it.

Reply:
ok